tests/fixtures/alchemy/get_nfts_for_contract.json

```diff
@@ -0,0 +1,293 @@
+{
+    "nfts": [
```

Comment:
general remark here as well, do we have any error states that can occur in the API responses? would be good to have tests for those as well

Reply:
You mean error states with non-200 status responses or responses with `raw.error` is not null? If I am not mistaken non-200 responses already handled in the `send` [method](https://github.com/ArdentHQ/dashbrd/blob/70ed02390f45f6d0056f10092c90bd31a04a02fd/app/Http/Client/Alchemy/AlchemyPendingRequest.php#L96), the responses with `raw.error` should already have it. Otherwise we would have coverage issue.

Reply:
yeah actual errors with `raw.error`. Main question is: are V3 error responses the same as V2 or not?

Reply:
if is the same, then good to go, just want confirmation of that

Reply:
![image](https://github.com/ArdentHQ/dashbrd/assets/132887516/a21339e5-2bee-4a40-b92c-673aaa262857)
According to spec the `error` key just moved into the `raw`, so I think we should be fine.